templates/template-scala-parallel-recommendation/README.md

```diff
@@ -0,0 +1,79 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Recommendation Template
+
+## Documentation
+
+Please refer to
+https://predictionio.apache.org/templates/recommendation/quickstart/.
+
+## Versions
```

Comment:
We will have to maintain README of each template after incorporating, I think it should be synchronized with PredictionIO release, but [text-classifier template](https://github.com/apache/predictionio-template-text-classifier) has inconsistent versioning with PredictionIO version. How should we handle that?

In addition, text-classifier template contains [large data](https://github.com/apache/predictionio-template-text-classifier/tree/develop/data) in its repository, I want to separate that data from the repository. Is there any suitable place to put them?

Reply:
I think the test you added to `check_templates.sh` is a good idea.
People will know that they have to change the corresponding template version while modifying PIO's version.
As for the data, maybe we add another non-test branch?